packages/dds/tree/src/simple-tree/schemaTypes.ts

```diff
@@ -224,14 +225,48 @@ export class FieldSchema<
 	protected _typeCheck?: MakeNominal;
 
 	/**
-	 * @param kind - The [kind](https://en.wikipedia.org/wiki/Kind_(type_theory)) of this field.
-	 * Determine the multiplicity, viewing and editing APIs as well as the merge resolution policy.
-	 * @param allowedTypes - What types of tree nodes are allowed in this field.
+	 * Lazily evaluated allowed types.
 	 */
+	private readonly lazyTypes: Lazy<AllowedTypes>;
+
+	/**
+	 * What types of tree nodes are allowed in this field.
+	 *
+	 * @remarks
+	 * This is the same set of types in {@link FlexFieldSchema.allowedTypes}, just as a set with
```

Comment:
you can't link internal docs from a public API. That needs to go in private remarks.

Reply:
Ah, right. Will update.